packages/block-library/src/navigation-menu/style.scss

```diff
@@ -61,13 +61,10 @@
 		& > li > ul {
 			margin: 0;
 			position: absolute;
-			background: #fff;
-			box-shadow: 0 0 8px 2px rgba(0, 0, 0, 0.2);
 			left: 0;
-			top: 100%;
+			top: calc(100% + 5px);
```

Comment:
I'm seeing a space between the submenu primary item, and the submenu, which is causing me to not be able to actually move the mouse there without the menu closing. I suspect it could be those 5px, which makes this for a tricky challenge with CSS-only menus.